packages/formation-react/src/components/Telephone/Telephone.unit.spec.jsx

```diff
@@ -27,24 +27,23 @@ describe('Widget <Telephone />', () => {
   it('should throw an error when number does not match pattern', () => {
     expect(() => {
       const wrapper = shallow(<Telephone />);
+      expect(wrapper.text()).to.equal('');
       wrapper.unmount();
-    }).to.throw('Contact number "" does not match the pattern (###-###-####)');
+    })
   });
   it('should throw an error when number is less than 10-digits', () => {
     expect(() => {
       const wrapper = shallow(<Telephone contact={4321} />);
+      expect(wrapper.text()).to.equal('4321');
       wrapper.unmount();
-    }).to.throw(
-      `Contact number "4321" does not match the pattern (###-###-####)`,
-    );
+    })
   });
   it('should throw an error when number is more than 10-digits', () => {
```

Comment:
this description is no longer correct :)

Reply:
Nice catch, thank you! 🙂